### PR TITLE
Windows installer: Rebuild and use pymavlink from sources if available

### PR DESCRIPTION
**Pull request by amilcarlucas**
If the user has a local copy of pymavlink chances are that he/she wants to use it :)

### CODE REVIEW

File: windows/MAVProxyWinBuild.bat [hunk 1]
```diff
@@ -21,7 +21,14 @@ rem -----build the changelog-----
 "%PYTHON_LOCATION%\python" createChangelog.py
 
 rem -----Upgrade pymavlink if needed-----
-"%PYTHON_LOCATION%\Scripts\pip" install pymavlink -U
+if exist "..\..\pymavlink" (
```
Comment (stephendade) [inline windows/MAVProxyWinBuild.bat]: Shouldn't this be "..\..\mavlink\pymavlink"? I suppose it depends on the user configuration - whether they're using the mavlink repository or pymavlink repository.

Comment (amilcarlucas) [inline windows/MAVProxyWinBuild.bat]: Yes, I will rebase this, with an "else if" for that case as well

**Comment by amilcarlucas** [comment]
I think now it has the features you wanted

**Comment by stephendade** [comment]
Thanks!

**Pull request merged**